tests/Unit/Classes/ToolsTest.php

```diff
@@ -474,7 +478,7 @@ public function providerMathRound(): array
      */
     public function testMathRound(float $expectedResult, float $value, int $precision, int $mode): void
     {
-        $this->assertSame($expectedResult, Tools::math_round($value, $precision, $mode));
+        $this->assertSame($expectedResult, Tools::ps_round($value, $precision, $mode));
```

Comment:
Why change this, the test is supposed to test the `math_round` method not the `ps_round` one